packages/xo-remote-parser/src/index.js

```diff
@@ -38,19 +38,32 @@ export const parse = string => {
     object.username = username
     object.password = password
   } else if (type === 's3' || type === 's3+http') {
-    const parsed = new Url(string)
+    const parsed = urlParser(string, true)
     object.protocol = parsed.protocol === 's3:' ? 'https' : 'http'
     object.type = 's3'
     object.region = parsed.hash.length === 0 ? undefined : parsed.hash.slice(1) // remove '#'
     object.host = parsed.host
     object.path = parsed.pathname
     object.username = parsed.username
     object.password = decodeURIComponent(parsed.password)
+    const qs = parsed.query
+    object.acceptSelfSigned = qs.acceptSelfSigned === 'true'
   }
   return object
 }
 
-export const format = ({ type, host, path, port, username, password, domain, protocol = type, region }) => {
+export const format = ({
+  type,
+  host,
+  path,
+  port,
+  username,
+  password,
+  domain,
+  protocol = type,
+  region,
+  acceptSelfSigned = false,
```

Comment:
Use the name `allowUnauthorized`, because:
- already used in other parts of XO
- more correct, this param does not only allow self signed certificate